pkg/pillar/hypervisor/kubevirt.go

```diff
@@ -0,0 +1,1421 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build kubevirt
+
+package hypervisor
+
+import (
+	"bufio"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"math"
+	"math/rand"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/lf-edge/eve/pkg/pillar/base"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+
+	netattdefv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	"github.com/lf-edge/eve/pkg/pillar/kubeapi"
+	"github.com/sirupsen/logrus"
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	metricsv "k8s.io/metrics/pkg/client/clientset/versioned"
+	v1 "kubevirt.io/api/core/v1"
+)
+
+// KubevirtHypervisorName is a name of kubevirt hypervisor
+const (
+	KubevirtHypervisorName = "kubevirt"
+	kubevirtStateDir       = "/run/hypervisor/kubevirt/"
+	eveLabelKey            = "App-Domain-Name"
```

Comment:
Do we refer to this App-Domain-Name anywhere else in EVE? Or we just want a unique string only used in this file?

Reply:
Only in this file.